packages/logan-center-plugin/src/ContentTypes/Project.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Klein\LoganCenter\ContentTypes;
+
+use Args\register_post_type;
+use Klein\LoganCenter\Concerns\OnInit;
+use Klein\LoganCenter\Concerns\RegistersTaxonomy;
+
+class Project
+{
+    use OnInit, RegistersTaxonomy;
+
+    /** Name/slug of the post type. */
+    const NAME = 'project';
+
+    /** Object types which can be connected to this object type. */
+    const CONNECTED_OBJECT_TYPES = ['post'];
+
+    public function __construct()
+    {
+        $this->setup_actions();
+    }
+
+    /**
+     * Get the set of arguments to pass to the object type registrar.
+     */
+    protected function get_registrar_args(): array
+    {
+        $args = new \Args\register_taxonomy;
+        $args->show_in_rest = true;
+        $args->description = 'Organized collections of individual reports investigating a facet of a Theme.';
```

Comment:
@bpizzle This is the description for the "Project" taxonomy to be displayed in the admin UI. It can be changed at any time since it's just helper text for the user, but open to feedback prior to merge.

Reply:
I tend to be a bit floofy in my wording, so it might even be better to remove the description entirely 🙃